### PR TITLE
MOVE-1148: Add informative tooltips to location markers in project view

### PR DESCRIPTION
**Pull request by imadsyed333**
# Issue Addressed
This PR closes MOVE-1148

# Description
Added a tooltip to location markers in the map of MOVE Project View. Tooltip displays study request ID, type, location, and duration. If there are multiple study requests at a single location, the tooltip will display them in a list. Each request also has a button to take the user to the Request page of that request.

image

# Tests
Tested in MOVE local v1.11.1 in Chrome and Firefox.

### CODE REVIEW

File: web/components/geo/map/FcMapPopup.vue [hunk 1]
```diff
@@ -81,6 +90,9 @@ export default {
     featureSelectable() {
       return SELECTABLE_LAYERS.includes(this.feature.layer.id);
     },
+    isStudyRequest() {
+      return !(this.feature.properties.studyRequests === undefined);
```
Comment (spencermountain) [inline web/components/geo/map/FcMapPopup.vue:94]: no biggie, i'd use `!==`

**Review by spencermountain — APPROVED** [review]
nice work!!

File: web/components/geo/map/FcMapPopup.vue (hunk 1 above)
Comment (imadsyed333) [reply web/components/geo/map/FcMapPopup.vue:94]: I'm now left baffled as to why I didn't do that 😅

**Review by ryanpdaley — APPROVED** [review]
I think this looks okay! There is something weird going on with the package-lock.json needing to be committed... but I am assuming this had to do with a fix that you tried for the lining issue... we'll see 👀

**Comment by imadsyed333** [comment]
> I think this looks okay! There is something weird going on with the package-lock.json needing to be committed... but I am assuming this had to do with a fix that you tried for the lining issue... we'll see 👀

Oh I think this might have been because I ran npm install to see if it would fix the linting issue (it didn't 😞 ).

**Pull request merged**